pandas/tests/io/parser/test_c_parser_only.py

```diff
@@ -646,7 +648,7 @@ def test_1000_sep_with_decimal(
     tm.assert_frame_equal(result, expected)
 
 
-@pytest.mark.parametrize("float_precision", [None, "high", "round_trip"])
+@pytest.mark.parametrize("float_precision", [None, "legacy", "high", "round_trip"])
```

Comment:
do we have a test that catches if in an invalid float_precision value is passed?

Reply:
- There is no such test
- Currently, in 1.1.1, if you give an incorrect value for `float_precision`, it treats it as `None`
```python
>>> df=pd.read_csv("../Misc/pandasDev/foo.csv", float_precision="yo")
>>> df
     A    B
0  NaN   B1
1  1.0  NaN
>>>

So I could add code to reject invalid values and then an appropriate test.  Let me know if I should do that.

Reply:
ok, can you add (and of course reject invalid values)